Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Stage: base image
-FROM node:18.18-bullseye-slim as base
+FROM node:20.8.0-bullseye-slim as base
```

Comment:
`20.8-bullseye-slim` so that patch-level gets picked up automatically? like in the CI tags above

Reply:
oh I thought I used that one, I'll check if that tag exist and change if it's there 👍

Reply:
i checked, it is!

Reply:
Done here: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/627/commits/3c9e0dc0bb765180c7198f08e70cee79fbe167ac

Reply:
you made it `20.8-bullseye` not `20.8-bullseye-slim` – is that deliberate?